framework/CodeInterfaces/Generic/GenericParser.py

```diff
@@ -22,6 +22,18 @@
 
 import os
 import sys
+import numpy as np
+
+def _reprIfFloat(value):
+  """
+    Uses repr if the value is a float
+    @ In, value, any, the value to convert to a string
+    @ Out, _reprIfFloat, string, a string conversion of this
+  """
+  if type(value) == float or type(value) == np.float64:
```

Comment:
I think there are utils that might be better for checking if "is a number", and might extend better. Additionally, maybe this would make a good utility method?

Reply:
Changed to isAFloat.